main.py

```
@@ -0,0 +1,245 @@
+            detail="Invalid API Key"
+        )
+    data = await request.json()
+    mg_id = data.get("mg_id")
+    con = duckdb.connect(os.getenv("DB_PATH"),read_only=True)
+    result = con.sql(f"select mg_id from {TABLE_NAMES['MG_HASH_MAP']} where mg_id like '{mg_id}%'").to_df()
+    return result['mg_id'].tolist()
+
+
+
+if __name__ == "__main__":
+    uvicorn.run("main:app", host="0.0.0.0", port=8000, workers=4)
```

Ruff (0.8.2)

Possible binding to all interfaces

(S104)